model_downloader/list_topologies.yml

```diff
@@ -1098,7 +1098,9 @@ topologies:
     license: https://github.com/lachinov/brats2018-graphlabunn/blob/master/LICENSE
 
   - name: "age-gender-recognition-retail-0013"
-    description: "Age & gender classification."
+    description: >-
+      Fully convolutional network for simultaneous Age/Gender recognition. The networkis able to recognize age of people in [18, 75]
```

Comment:
This looks much better than before, but it's still a few characters away from fitting into GitHub's diff view. Could you decrease the line width by just a little bit more?

Reply:
@IRDonch, widht is 80 now, is it enough?

Reply:
Yeah, this looks much better.